model/move.ts

```diff
@@ -0,0 +1,16 @@
+import { match, P } from "ts-pattern";
+
+export type Move = "Rock" | "Paper" | "Scissors";
```

Comment:
👍 This is the way

Enums are considered a legacy feature in TypeScript, and it's nowawadays common practice using string literals to represent enums.

If you need these values at runtime, you can also use this pattern:

```tsx
const Move = {
  Rock: null,
	Paper: null,
	Scissors: null
}
type Move = keyof Move;
```